test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -200,6 +200,8 @@ protected void afterIfFailed(List<Throwable> errors) {
                 Files.walk(path).forEach(file -> {
                     if (file.toString().endsWith(".yml")) {
                         addSuite(root, file, files);
+                    } else if (file.toString().endsWith(".yaml")) {
+                        throw new IllegalArgumentException("yaml files are no longer supported: " + file);
```

Comment:
ok I wonder what those txt files are, but then LGTM, I agree that we should get this in meanwhile.